front50-azure/src/main/java/com/netflix/spinnaker/front50/model/AzureStorageService.java

```diff
@@ -104,31 +104,6 @@ public boolean supportsVersioning() {
   }
 
   @Override
-  public <T extends Timestamped> Collection<T> loadObjectsWithPrefix(ObjectType objectType, String prefix, int maxResults) {
```

Comment:
This was never called by a controller method and has been removed (similar changes to other `StorageService` implementations).